align_anything/serve/chatbot.py

```diff
@@ -82,6 +84,8 @@ class SpecialCommand(Enum):
     EXIT = '/exit: End the dialogue and quit.'
     REGENERATE = '/regenerate: Regenerate the last response.'
     HELP = '/help: Show this help message.'
+    IMAGE = '/image: Select an image as input.'
+    VLLM = '/vllm: Whether to use VLLM model'
```

Comment:
VLLM is a fast and easy-to-use library for LLM inference and serving while VLM is the Vision Language Model. Please rename this config.